scripts/utils/delete_subscriptions.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python3
+# Copyright 2024 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+import requests
+import sys
+
+requests.packages.urllib3.disable_warnings()
+
+__author__ = 'Md Arqum Farooqui'
+
+##############################################################
+# BEGIN of the configuration part (don't touch above this line ;)
+
+cb_endpoint = 'http://localhost:1026/v2/subscriptions'
+
+headers = {
+    'Accept': 'application/json',
+    'fiware-service': 'service',
+    'fiware-servicepath': '/subservice'
+}
+
+page_size = 500
+
+# WARNING! use this filter or you will remove *all* subscriptions
+filter = 'offset=0'
+
+# END of the configuration part (don't touch below this line ;)
+##############################################################
+
+def delete_subscriptions():
+    """
+    delete subscriptions in current page.
+    :return: True if removal was ok, False otherwise
+    """
+    try:
+        # Fetching subscriptions with pagination
+        response = requests.get(f"{cb_endpoint}?limit={page_size}&{filter}", headers=headers, verify=False)
```

Comment:
```suggestion
        response = requests.get(f"{cb_endpoint}?limit={page_size}{filter}", headers=headers, verify=False)
```

Let's assume that filter is either the empty string or a query param with the `&` already included (as in existing delete_entities_by_query.py)

Reply:
fixed in [01f8553](https://github.com/telefonicaid/fiware-orion/pull/4626/commits/01f855348599b3e69c8672ac89e28656b2e51a20)